### PR TITLE
Feat/3016 generated app =>  update readme files for the server and the admin-ui

### PR DESCRIPTION
**Pull request by morhag90**
Issue Number: #3017

## PR Details

After the split of the docker files we needed to update readme files for the server and the admin-ui with new commands and add the env variables list 

## PR Checklist
- [ X] Tests for the changes have been added
- [ X] `npm test` doesn't throw any error

### CODE REVIEW

File: packages/amplication-data-service-generator/src/server/static/README.md [hunk 1]
```diff
@@ -24,36 +24,75 @@ Your app is ready to be deployed!
...
+
 - Install dependencies
+- Move to server directory
+
+```
+
+  cd server
 
 ```
+
```
Comment (mshidlov) [inline packages/amplication-data-service-generator/src/server/static/README.md:50]:
```suggestion
```
```

Comment (mshidlov) [reply packages/amplication-data-service-generator/src/server/static/README.md:50]: image

File: packages/amplication-data-service-generator/src/server/static/package.json [hunk 2]
```diff
@@ -15,7 +15,9 @@
     "db:init": "run-s \"db:migrate-save -- --name 'initial version'\" db:migrate-up seed",
     "prisma:generate": "prisma generate",
     "docker:db": "docker-compose -f docker-compose.db.yml up -d",
-    "docker:build": "docker build ."
+    "docker:build": "docker build .",
+    "docker": "docker-compose up -d",
+    "docker:down": "docker-compose down --volumes"
```
Comment (mshidlov) [inline packages/amplication-data-service-generator/src/server/static/package.json:20]:
```suggestion
    "compose:up": "docker-compose up -d",
    "compose:down": "docker-compose down --volumes"
```

**Comment by mshidlov** [comment]
@morhag90 I'm missing the Admin UI readme

**Review by mshidlov — CHANGES_REQUESTED** [review]
please see my comments

File: packages/amplication-data-service-generator/src/server/static/README.md (hunk 1 above)
Comment (morhag90) [reply packages/amplication-data-service-generator/src/server/static/README.md:50]: done

**Review by mshidlov — CHANGES_REQUESTED** [review]

File: packages/amplication-data-service-generator/src/server/static/README.md [hunk 3]
```diff
@@ -24,34 +24,74 @@ Your app is ready to be deployed!
...
 
 - Start database in Docker
 
 ```
+
 npm run docker:db
+
+```
+
+- Or, instead running db only, run docker with server
```
[the remark this answers is not in the document]
Comment (morhag90) [reply packages/amplication-data-service-generator/src/server/static/README.md:73]: @mshidlov something like that?:
image

[the remark this answers is not in the document]
Comment (mshidlov) [reply packages/amplication-data-service-generator/src/server/static/README.md]: @morhag90 
Set up development environment 
db:init
npm start

Run server deployment (docker-compose)

File: packages/amplication-data-service-generator/src/admin/static/README.md [hunk 4]
```diff
@@ -5,42 +5,40 @@ You can learn more in the [Amplication documentation](https://docs.amplication.c
...
+
+| Environment          | Description                                 | Value                 |
+| -------------------- | ------------------------------------------- | --------------------- |
+| PORT                 | The port that the client UI is listening to |
+| REACT_APP_SERVER_URL | Amplication Server URL                      | http://localhost:3000 |
+
 ## Available Scripts
 
-In the project directory, you can run:
+In the `admin-ui` subdirectory, you can run:
```
Comment (yuval-hazaz) [inline packages/amplication-data-service-generator/src/admin/static/README.md:17]: This should be 3001

File: packages/amplication-data-service-generator/src/server/static/README.md [hunk 5]
```diff
@@ -2,58 +2,84 @@
...
+| -------------------- | ---------------------------------------- | ----------------------------------------------------------- |
+| DEBUG_MODE           | Debug level                              | 1                                                           |
+| POSTGRESQL_URL       | Local database connection URL            | postgresql://admin:admin@localhost:5432/\${SERVICE_DB_NAME} |
+| POSTGRESQL_PORT      | Local database port                      | 5432                                                        |
+| POSTGRESQL_USER      | Local database username                  | admin                                                       |
+| POSTGRESQL_PASSWORD  | Local database password                  | admin                                                       |
+| COMPOSE_PROJECT_NAME | Docker Compose project name              | amp\_{applicationId}                                        |
+| SERVER_PORT          | The port that the server is listening to | 3000                                                        |
+| JWT_SECRET_KEY       | JWT secret                               | XAFzBpM3es                                                  |
```
Comment (yuval-hazaz) [inline packages/amplication-data-service-generator/src/server/static/README.md:33]: I dont think we should use a real password - it should be clear to the user that this is an example they need to change - so using something like 123456 or changMe is better

**Review by yuval-hazaz — COMMENTED** [review]
@morhag90 please see my comments

**Pull request merged**